spec/team_spec.rb

```diff
@@ -24,12 +24,30 @@
     end
   end
 
-  describe "methods" do
+  describe '#methods' do
     it 'can create Team objects using the create_from_csv method' do
       expect(@teams.first.id).to eq "1"
       expect(@teams.first.name).to eq "Atlanta United"
     end
 
+    it '#highest_scoring_visitor outputs correctly' do
+      expect(Team.highest_scoring_visitor).to eq "FC Dallas"
+    end
+
+    it '#lowest_scoring_visitor outputs correctly' do
+      expect(Team.lowest_scoring_visitor).to eq "San Jose Earthquakes"
+    end
+
+    it '#generate_score_data creates correct sorted data' do
```

Comment:
Thanks for adding tests for specific return values. I feel super confident in the method now.